abis-supplemental/IDependencyRegistryV0.json

```diff
@@ -0,0 +1,350 @@
+{
```

Comment:
had to add this because updated contract abi has not yet been incorporated to the subgraph codebase. this is the previous npm package release to decouple that work.